roofit/roofit/inc/RooGaussExpTails.h

```diff
@@ -0,0 +1,48 @@
+#ifndef RooFit_RooFit_RooGaussExpTails_h
+#define RooFit_RooFit_RooGaussExpTails_h
+
+#include "RooAbsPdf.h"
+#include "RooRealProxy.h"
+
+class RooAbsReal;
+
+class RooGaussExpTails : public RooAbsPdf {
+public:
+  RooGaussExpTails() { };
+  RooGaussExpTails(const char *name, const char *title,
+                         RooAbsReal::Ref _x,
+                         RooAbsReal::Ref _x0,
+                         RooAbsReal::Ref _sigma,
+                         RooAbsReal::Ref _kL,
+                         RooAbsReal::Ref _kH
+                         );
+  RooGaussExpTails(const RooGaussExpTails& other, const char* name=nullptr);
+  TObject* clone(const char* newname) const override {
+    return new RooGaussExpTails(*this,newname);
+  }
+
+  Int_t getAnalyticalIntegral(RooArgSet& allVars, RooArgSet& analVars, const char* rangeName=nullptr) const override;
+  double analyticalIntegral(Int_t code, const char* rangeName=nullptr) const override;
+
+  RooAbsReal::Ref x() const { return *x_; }
+  RooAbsReal::Ref x0() const { return *x0_; }
+  RooAbsReal::Ref sigma() const { return *sigma_; }
+  RooAbsReal::Ref kL() const { return *kL_; }
+  RooAbsReal::Ref kH() const { return *kH_; }
```

Comment:
```suggestion
  RooAbsReal const &x() const { return *x_; }
  RooAbsReal const &x0() const { return *x0_; }
  RooAbsReal const &sigma() const { return *sigma_; }
  RooAbsReal const &kL() const { return *kL_; }
  RooAbsReal const &kH() const { return *kH_; }
```
These getters should return normal references (the `RooAbsReal::Ref` is only meant for the constructors, to implicitly create RooFit constants if one passes a `double` instead of a `RooAbsReal`).